packages/api-queries/src/me-notifications-settings.ts

```diff
@@ -11,6 +14,27 @@ export const userNotificationsSettingsQuery = () =>
 		queryFn: fetchUserNotificationSettings,
 	} );
 
+export const userNotificationsSettingsMutation = () =>
```

Comment:
@Addison-Stavlo, I created the CML-913 to unify this mutation with the userNotificationsExtrasSettingsMutation.